examples/with-typescript/README.md

```diff
@@ -18,29 +20,33 @@ Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_mediu
 
 Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
 
-```bash
-npx create-next-app --example with-typescript with-typescript-app
+```sh
+npx create-next-app --typescript with-typescript-app
 # or
-yarn create next-app --example with-typescript with-typescript-app
+yarn create next-app --typescript with-typescript-app
 ```
 
 Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=next-example) ([Documentation](https://nextjs.org/docs/deployment)).
 
 ## Notes
 
-This example shows how to integrate the TypeScript type system into Next.js. Since TypeScript is supported out of the box with Next.js, all we have to do is to install TypeScript.
+This example shows how to integrate the TypeScript type system into Next.js. Since TypeScript is supported out of the box with Next.js, all we have to do is to install TypeScript. (Installing this template will do this for you)
```

Comment:
```suggestion
This example shows how to integrate the TypeScript type system into Next.js. Since TypeScript is supported out of the box with Next.js, all you have to do is to get started is to use this example. Alternatively, you can [install TypeScript manually](https://nextjs.org/docs/basic-features/typescript#existing-projects) to an existing project. 
```

A suggestion, what we're trying to say is, you can get started with TypeScript by using this example or the CLI but you can also add TypeScript to your existing project - link to docs.